scripts/zones/Apollyon/IDs.lua

```diff
@@ -188,6 +168,69 @@ zones[xi.zone.APOLLYON] =
         APOLLYON_CS_CRATE = 16933126,
         APOLLYON_CENTRAL_CRATE = 16933123,
     },
+    SE_APOLLYON =
+    {
+        mob =
```

Comment:
The mob side of these could probably be moved to dynamic lookups (I'd like to see npcs as well, but there's an open issue that needs to be resolved with load order)

Reply:
From what I can see it has to be in the format of `ID.mob.<mob_name> = DYNAMIC_LOOKUP` and then it'll find it. It doesn't appear like it will work with mobs that have the same name, is that right? It would be nice to use this but I think the feature needs to be expanded upon first.

Reply:
The only realistic way you could do this is by looking up Tieholtsodi then using that ID plus an offset for the adamantshells. There might be something similar before or after the flying spears

Reply:
Yeah, I talked to @claywar about how this could be improved. I may put together a PR for it. Got lots to do!